hasura/metadata/tables.yaml

```diff
@@ -343,6 +400,169 @@
       - imageUrl
       - title
       filter: {}
+- table:
+    schema: public
+    name: quest
+  object_relationships:
+  - name: guild
+    using:
+      foreign_key_constraint_on: guild_id
+  - name: player
+    using:
+      foreign_key_constraint_on: created_by_player_id
+  - name: QuestRepetition
+    using:
+      foreign_key_constraint_on: repetition
+  - name: QuestStatus
+    using:
+      foreign_key_constraint_on: status
+  array_relationships:
+  - name: quest_completions
+    using:
+      foreign_key_constraint_on:
+        column: quest_id
+        table:
+          schema: public
+          name: quest_completion
+  - name: quest_skills
+    using:
+      foreign_key_constraint_on:
+        column: quest_id
+        table:
+          schema: public
+          name: quest_skill
+  select_permissions:
+  - role: player
+    permission:
+      columns:
+      - created_at
+      - cooldown
+      - description
+      - external_link
+      - repetition
+      - status
+      - title
+      - created_by_player_id
+      - guild_id
+      - id
+      filter: {}
+  - role: public
+    permission:
+      columns:
+      - id
+      - created_at
+      - guild_id
+      - created_by_player_id
+      - title
+      - description
+      - external_link
+      - cooldown
+      - status
+      - repetition
+      filter: {}
+  update_permissions:
+  - role: player
+    permission:
+      columns:
+      - cooldown
+      - description
+      - external_link
+      - repetition
+      - status
+      - title
+      filter:
+        created_by_player_id:
+          _eq: X-Hasura-User-Id
+      check: null
```

Comment:
what is `check`?

Reply:
I don't know, this is auto generated by hasura. Maybe it is the "post-update" check.

Reply:
Yep thats what it is